packages/headless/src/controllers/insight/did-you-mean/headless-insight-did-you-mean.ts

```diff
@@ -36,13 +41,14 @@ export type {
  */
 export function buildDidYouMean(
   engine: InsightEngine,
-  props: DidYouMeanProps = {
-    options: {
-      queryCorrectionMode: 'legacy',
-    },
-  }
+  props: DidYouMeanProps
```

Comment:
That's a breaking change actually to remove the default value passed to props.

cause if my implementation, I used to call `buildDidYouMean(engine)`,  with this PR, my implementation will break as there no longer default values for props, I will get error like `buildDidYouMean expects two arguments, got one`